WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/dashboard/todaysstats/TodaysStatsCardViewHolder.kt

```diff
@@ -99,23 +95,6 @@ class TodaysStatsCardViewHolder(
         )
     }
 
-    private fun SpannableString.withExternalLinkImageSpan(endIndex: Int) {
-        val drawable = ContextCompat.getDrawable(itemView.context, drawable.ic_external_white_24dp) ?: return
-        drawable.setTint(linkColor)
-        drawable.setBounds(
-                EXTERNAL_LINK_DRAWABLE_BOUND_LEFT,
-                0,
-                drawable.intrinsicWidth / 2,
-                drawable.intrinsicHeight / 2
-        )
-        setSpan(
-                ImageSpan(drawable, ImageSpan.ALIGN_BASELINE),
-                endIndex,
-                endIndex + 1,
-                Spannable.SPAN_INCLUSIVE_EXCLUSIVE
-        )
-    }
-
```

Comment:
This code was removed via https://github.com/wordpress-mobile/WordPress-Android/pull/16446 to fix a crash found last-minute during my smoke-testing of the app before submission.